docs/sources/alerting/alerting-rules/manage-contact-points/integrations/configure-oncall.md

```diff
@@ -6,6 +6,8 @@ keywords:
   - alerting
   - oncall
   - integration
+aliases:
+  - ./integrations/configure-oncall/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/configure-oncall/
```

Comment:
@jdbaldry can you help with this alias? I can't get it work in my local. I've tried a couple different ways (no ./ at the beginning) and I'm stumped.

Reply:
Fixed in https://github.com/grafana/grafana/pull/78549/commits/cf5cc17dfde8aef4ff378c7dea59426aa74820af